core/services/head_tracker.go

```diff
@@ -39,18 +40,15 @@ var (
 		Help:    "How long it took to execute all callbacks (ms) histogram",
 		Buckets: []float64{50, 100, 250, 500, 1000, 2000, 5000, 10000, 15000, 30000, 100000},
 	})
+
+	// kovanChainID is the Chain ID for Kovan test network
+	kovanChainID = big.NewInt(42)
```

Comment:
It is a constant for Kovan which identifies that chain.

> Why do chains have an ID? 

To prevent replay attacks, e.g. if I send a transaction on ropsten, the chain ID prevents this transaction being replayed on mainnet.

Eth nodes cannot support multiple chains - they are pegged to just one.